test/unit-tests/ad_manager_utils_test.js

```diff
@@ -0,0 +1,94 @@
+/*
+ * Unit test class for the Vast Ad Manager
+ * https://github.com/Automattic/expect.js
+ */
+
+var adManagerUtils = require("../../utils/ad_manager_utils.js");
+
+describe('Ad Manager Utility Class:', function()
+{
+
+  it('Time Stamps Should Be Converted to Milliseconds', function()
```

Comment:
I originally did not add the unit tests for improper hh:mm:ss formats because the caller of the utility function (ad_manager_vast.js) does regex filtering for those improper formats. But yes, I'll add checks for improper hh:mm:ss to the function for the sake of this utility class being generic.